app/v1/dbControllers/user.js

```diff
@@ -0,0 +1,30 @@
+import pool from '../dbModel/connection';
+
+class UserController {
+	static signup(request, response) {
+		const {
+			firstname, lastname, username, phoneNumber, email, password
+		} = request.body;
+		const selectQuery = {
+			text: 'INSERT INTO users (firstname, lastname, username, phoneNumber, email, password, isadmin) VALUES($1, $2, $3, $4, $5, $6, $7) RETURNING *',
+			values: [firstname, lastname, username, phoneNumber, email, password, false]
+		};
+		pool.query(selectQuery).then((users) => {
+			console.log(users);
+			if (users.rows) {
+				return response.status(201).json({
+					status: 201,
+					message: true,
+					data: [users.rows[0]]
+				});
+			}
+		}).catch(error => (
+			response.status(400).json({
```

Comment:
Ideally, before saving a new user, you could run a query to see if they already exist and return a 409 if they do. Then you would return a 500 at this point which would signify that something went wrong